### DISCUSSION TITLE
Which is the recommended practice for creating nested routes in Nuxt 3?

### DISCUSSION THREAD

**Discussion by jhoryong** [nuxt/nuxt#19705] [Questions]
I've read nuxt3 documentations for making nested routes, and below is the presented way of making nested routes.

-| pages/
---| parent/
------| child.vue
---| parent.vue

If "parent" page has its own route like '/parent', it has its component outside the parent folder.
However, in some nuxt3 tutorial video on Youtube, I could see the structure below, which creates index.vue under the parent folder.

-| pages/
---| parent/
------| index.vue
------| child.vue

I am new to nuxt so I don't know the practices in nuxt2, but maybe that's the past pattern and it gave me a bit of confusion.
For using nuxt3, is nuxt3 doc way (the top one) the recommended way for nested routes?
Thank you in advance!

**Comment by ndragun92** [comment]
I always do something like
```
/pages
/pages/parent/index.vue
/pages/parent/child/index.vue
/pages/parent/child/[id]/index.vue
etc...
```

But I guess it all depends on team as well and how are you satisfied with structure you build.
